src/utils/runWorkspaceTasks.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import type Task from '../Project';
+import Project from '../Project';
+
+export default async function runWorkspaceTasks(task: Task) {
```

Comment:
I'm not sure whether this is a right approach, but if we don't want to expose Project's API it looks fine to me. Also we can keep Project.runWorkspaceTasks api unchanged.